qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -700,6 +700,12 @@ def _init_js(self):
         self._inject_early_js('js', js_code, subframes=True)
         self._init_stylesheet()
 
+        js_code_print = utils.read_file('javascript/print.js')
```

Comment:
No need for a separate variable here - just call `utils.read_file` directly below.